Building/Plank.tscn

```diff
@@ -9,6 +9,7 @@ extents = Vector2( 80, 8 )
 [node name="Plank(RigidBody2D)" type="RigidBody2D"]
 collision_layer = 2
 collision_mask = 2
+contacts_reported = 3
```

Comment:
We should probably turn this back off, as we're using proximity rather than collision